src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let totalSum = 0;
+
+  const adder = (a) => {
```

Comment:
The `adder` function does not check if the input `a` is a number. While the task description states that there's no need to check if the argument is a number, be aware that passing a non-numeric value will result in unexpected behavior.